### ISSUE TITLE
Add macOS app builds to CircleCI config

### ISSUE THREAD

**Issue by zone117x** [blockstack/blockstack-browser#1811]
@zone117x commented on [Thu Jan 24 2019](https://github.com/blockstack/gaia/issues/193)

**Comment by zone117x** [comment]
Simple TravisCI config example https://travis-ci.org/zone117x/blockstack-browser/jobs/481526427

**Comment by markmhx** [comment]
@zone117x Want to tackle this in the new sprint starting today?

**Comment by zone117x** [comment]
Yep. This task is pretty easy once https://github.com/blockstack/blockstack-browser/pull/1803 gets merged.

**Comment by markmhx** [comment]
We discussed that we should build unsigned builds for now
